app/src/main/java/com/wootric/androidsdk_demo/MainActivity.java

```diff
@@ -2,11 +2,12 @@
 
 import android.app.Activity;
 import android.os.Bundle;
+import android.support.v4.app.FragmentActivity;
 import android.view.View;
 
 import com.wootric.androidsdk.Wootric;
 
-public class MainActivity extends Activity {
+public class MainActivity extends FragmentActivity {
```

Comment:
Don't really need to modify this. But if you can do it if you want to. Take it or leave it...